incubator/swift/image/project/Dockerfile

```diff
@@ -10,6 +10,24 @@ RUN apt-get update \
 # Install user-app dependencies
 WORKDIR "/project/user-app"
 COPY ./user-app ./
-RUN swift build -c release
+RUN echo "#!/bin/bash" > run.sh
```

Comment:
Can we merge these into one `RUN` statement, otherwise each line generates a new layer in the image.

```dockerfile
RUN echo "#!/bin/bash" > run.sh \
  && swift build -c release | awk 'END {print $NF}' >> run.sh \
  && chmod 755 run.sh
```

?

Reply:
I've merged it into a single step, and tested the changes.